### ISSUE TITLE
Genome alterations on trees

### ISSUE THREAD

**Issue by jvamosi** [OpenTreeOfLife/hackathon#8]
Sorry for this newbie post. I'm not a coder but an avid end-user. Perpetual issues have come up with polyploidization. Ploidy information and genome size are important traits for mapping on trees and influence the extent to which geography is involved in diversification. Having both geographical information and ploidy could help us understand how the two factors are involved in hybridization and polyploidization. Importantly, allopolyploidization and hybridization influence the construction of phylogenies themselves and overlay reticulate structure and this complication is often not considered in generating the trees themselves. A rather huge undertaking I imagine, but can the TreeofLife have functions for sorting out paralogs from orthologs, or build on the foundations of chromEvol (http://www.tau.ac.il/~itaymay/cp/chromEvol/) to examine the influences of major genome alterations in the Tree of Life?

**Comment by msrosenberg** [comment]
It seems like there are a couple of interesting ideas here. Please correct me if I misunderstand any of them.
1. Generally, being able to map some external "character" data onto the tree, or put another way, extracting a subtree and automatically mapping desired data onto that tree
2. Explicitly having DB's have genome size and ploidy. These certainly aren't in the purview of the OToL project, but would be useful resources to have and connect to.
3. Ortholog/paralog identification. I think the OToL is focused on species trees, not gene trees, so to identify ortholog/paralogs, we'd need a way of taking an external gene tree, comparing it to the OToL and determining ortholog/paralog status from that comparison. I suspect there are already algorithms/programs to do this generally, but we'd need to link the OToL into the process.

Definitely interesting ideas to consider.

**Comment by jvamosi** [comment]
I don't think you've misunderstood anything. With regard to orthologs/paralogs, I think the level of disagreement among trees will be closely related with the amount of "wackiness" (technical term) going on in the genome. In plants at least, the number of locations in the tree where the species are different ploidies is substantial, to the point where any confidence in reconstructing the evolutionary histories of constituent species is severely marred. I think it would be great if the OtoL could even just highlight where this lunacy is occurring. Can you make those branches glow with pretty colours perhaps? Like warning beacons?

**Comment by katieMlyons** [comment]
It would be interesting to be able to visualize the shakiest parts of the tree, based either on ploidy (or some other trait) or on discordance among submitted trees.

**Comment by chinchliff** [comment]
Yes, I think there are multiple targets here, as Mike pointed out above. It sounds like this might also overlap a little with issues like #17.
